Documentation/installation/requirements-aks.rst

```diff
@@ -7,8 +7,14 @@ BYOCNI (``none``)             Encapsulation   Cluster Pool        Kubernetes CRD
 Legacy Azure IPAM (``azure``) Direct Routing  Azure IPAM          Kubernetes CRD
 ============================= =============== =================== ==============
 
-Using `Bring your own CNI <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`_
-is the preferred way to run Cilium on `Azure Kubernetes Service (AKS) <https://docs.microsoft.com/en-us/azure/aks/>`_,
-however integration with the Azure stack via the :ref:`Azure IPAM<ipam_azure>`
-is not available and will only work with clusters not using BYOCNI. While still
-maintained for now, this mode is considered legacy.
+The preferred way to run Cilium on `Azure Kubernetes Service (AKS) <https://docs.microsoft.com/en-us/azure/aks/>`_ is 
+either `Bring your own CNI <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`_ or 
+`Azure CNI Powered by Cilium <https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium>`_. 
+Choosing between these options depends on your specific requirements and use case. 
+If you need full control over Cilium and its features, then Bring your own CNI approach might be preferred. 
+On the other hand, if you prefer a managed solution with simplified installation and 
+integration with Azure Virtual Network stack, then Azure CNI Powered by Cilium option might be suitable.
```

Comment:
Out of curiosity, what does "integration with Azure Virtual Network stack" mean? Is this referring to native IPAM from Azure?

Reply:
yes that's right.